src/components/Modal.js

```diff
@@ -23,27 +23,29 @@ const Modal = ({
         <div className="modal-main-title">
           <h1>{title}</h1>
         </div>
-        <div className="modal-main-content">
-          <p>{body}</p>
-        </div>
-        <div className="modal-main-buttons">
-          <button
-            className={`button ${
-              !secondaryButtonAction && !secondaryButtonText
-                ? "cta-button secondary"
-                : "cta-button primary"
-            }`}
-            onClick={secondaryButtonAction || handleClose}
-          >
-            {secondaryButtonText || "Cancel"}
-          </button>
-          <button
-            className="button cta-button primary"
-            onClick={primaryButtonAction}
-          >
-            {primaryButtonText || "Ok"}
-          </button>
-        </div>
+        <div className="modal-main-content">{body}</div>
+        {primaryButtonAction !== undefined ? (
+          <div className="modal-main-buttons">
+            <button
+              className={`button ${
+                !secondaryButtonAction && !secondaryButtonText
+                  ? "cta-button secondary"
+                  : "cta-button primary"
+              }`}
+              onClick={secondaryButtonAction || handleClose}
+            >
+              {secondaryButtonText || "Cancel"}
+            </button>
+            <button
+              className="button cta-button primary"
+              onClick={primaryButtonAction}
+            >
+              {primaryButtonText || "Ok"}
+            </button>
+          </div>
+        ) : (
+          ""
+        )}
       </div>
```

Comment:
I think you can undo the changes to this file. We won't have any modals without action buttons.

Reply:
Done :-)